media/css/mozorg/home/home-mr1-promo.scss

```diff
@@ -0,0 +1,302 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+
+
+// Firefox MR1 promo
+
+.c-fxpromo {
+    margin: 0 auto;
+    overflow: hidden;
+}
+
+.c-fxpromo-title {
+    @include font-firefox;
+    @include text-title-xl;
+    color: $color-ink-80;
+    text-align: center;
+}
+
+.mzp-c-button-download-container {
+    display: block;
+    margin: 0 auto $layout-md;
+}
+
+.c-fxpromo-altcta {
+    display: none;
+    margin: 0 auto $layout-md;
```

Comment:
The text appears more vertically centered next to the image if there is no bottom margin. OTOH, maybe putting the text a bit higher up is intentional, in which case ignore this.